### PR TITLE
Ticket 1 and Ticket 10 Critical

### LINKED ISSUE #1: Cart resource has duplicate line items
When a client requests the `/profile/cart` resource, the following two keys are on the response object.

1. `lineitems`
1. `line_items`

The `line_items` key needs to be removed from the response.

### LINKED ISSUE #10: Division by zero error on products
# CRITICAL ISSUE

When the client requests a single product, or all products, the status code is 500 and the response is always the following.

```
ZeroDivisionError at /products
division by zero
```

The most recent feature added by the team was adding an average rating key to each product. You might want to start there and see if there are any `try/catch` blocks you can implement.

### PR DESCRIPTION
**Pull request by Kamiran79**
Description of PR that completes issue here...

## Changes

- Item 1: line 65 file models/product.py add the try/except with return -1 if have zero division error
- Item 2: line 159 commented because it will return extra line items which we already pulled from Order Serializer

## Requests / Responses

If this PR contains code that defines a new request/response, or changes an existing one, please put the JSON representations here.

**Request**

GET `/profile/cart` get list of product added to the cart

```json
{
    "id": 2,
    "url": "http://localhost:8000/orders/2",
    "created_date": "2019-04-12",
    "payment_type": null,
    "customer": "http://localhost:8000/customers/7",
    "lineitems": [
        {
            "id": 4,
            "product": {
                "id": 52,
                "name": "900",
                "price": 1296.98,
                "number_sold": 0,
                "description": "1987 Saab",
                "quantity": 2,
                "created_date": "2019-03-19",
                "location": "Vratsa",
                "image_path": null,
                "average_rating": -1
            }
        },
        {
            "id": 5,
            "product": {
                "id": 33,
                "name": "Stratus",
                "price": 1199.91,
                "number_sold": 0,
                "description": "2001 Dodge",
                "quantity": 1,
                "created_date": "2019-04-06",
                "location": "Tianning",
                "image_path": null,
                "average_rating": -1
            }
        },
        {
            "id": 6,
            "product": {
                "id": 71,
                "name": "Sebring",
                "price": 1045.66,
                "number_sold": 0,
                "description": "1999 Chrysler",
                "quantity": 4,
                "created_date": "2019-05-18",
                "location": "Namibe",
                "image_path": null,
                "average_rating": -1
            }
        }
    ],
    "size": 3
}
```

**Response**

GET /profile/cart HTTP/1.1" 200 777 OK

## Testing

Description of how to test code...
Run Get request from Postman with this link "http://localhost:8000/profile/cart"

I got error ZeroDividsionError: division by zero
"GET /profile/cart HTTP/1.1" 500

- [ ] Fix Zero Division error
- [ ] Run test OK
- [ ] Fixed duplicate line items for #1
- [ ] Noticed after fixed first issue, the #10 was pointing for the Division zero and was critical ticket. I am going back to backlog and check all any critical ticket market as critical priority and add them to my list

## Related Issues

- Fixes #1
- Fixes #10

### CODE REVIEW

File: bangazonapi/models/product.py [hunk 1]
```diff
@@ -61,9 +62,13 @@ def average_rating(self):
         total_rating = 0
         for rating in ratings:
             total_rating += rating.rating
+        try:
+            avg = total_rating / len(ratings)
+            return avg
+        except:
```
Comment (hannahhall) [inline bangazonapi/models/product.py]: Can you add the name of the exception here? It's better to be explicit about which exception the code is catching

**Review by hannahhall — CHANGES_REQUESTED** [review]

**Comment by Kamiran79** [comment]
- line 68 I add the name of the exception

**Review by Kamiran79 — COMMENTED** [review]
Add the name of the exception

**Review by hannahhall — APPROVED** [review]
👍

**Pull request merged**